src/test/java/com/browserstack/util/Utility.java

```diff
@@ -0,0 +1,59 @@
+package com.browserstack.util;
+
+import org.json.simple.JSONObject;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+public class Utility {
+
+    private Utility(){}
+
+    public static Map<String,String> getLocalOptions(JSONObject config) {
+        Map<String, String> localOptions = new HashMap<>();
+        JSONObject localOptionsJson = (JSONObject) ((JSONObject) ((JSONObject) config.get("tests")).get("local")).get("local_binding_caps");
```

Comment:
considering local to be bstack_local in the config file too. it is a change across. thinking if we should still do it. what do you say?